Framework/Foundation/include/Framework/Endian.h

```diff
@@ -15,10 +15,16 @@
 // Lookup file for __BYTE_ORDER
 #ifdef __APPLE__
 #include <machine/endian.h>
+#define swap16_ ntohs
+#define swap32_ ntohl
+#define swap64_ ntohll
```

Comment:
Why do you actually need the aliases? Can't you simply stick to the POSIX nomenclature?

Reply:
No, I need uniform definitions here, both apple and gcc have aliases to builtin functions for swapping, only apple ones are mapped to `ntoh/hton`.

Reply:
all you need  is to define ntohll on linux, no? ntohl and ntohs are POSIX.1-2001 and are available on both mac and linux.

Reply:
No, these functions are somewhere in network headers on linux, while the builtins are gcc's own, which is simpler to use.

Reply:
I disagree, the header in question is quite minimal and in any case it would end up in a couple of CXX files. Anyways, we can fix it later.